simphony_ui/global_parameters_model.py

```diff
@@ -0,0 +1,17 @@
+from traits.api import HasStrictTraits, Int, Enum
+from traitsui.api import View, Item, VGroup
+
+
+class GlobalParametersModel(HasStrictTraits):
+    """ The model of Global input parameters """
+
+    num_iterations = Int(10)
```

Comment:
A style comment: if you do a comment like
```
    #: The number of iterations in the simulation.
    num_iterations = Int(10)
```
then auto-documentation tools can pick that up and associate it to the trait.